lib/test_nplusone.py

```diff
@@ -54,20 +55,36 @@ def worker0(step):
 
     step(2,'Add %s files and check if we still have k1+nfiles after resync'%nfiles)
 
+    total_size=0
+    sizes=[]
+
+    # compute the file sizes in the set
     for i in range(nfiles):
-        create_hashfile(d,size=filesize)
+        size=size2nbytes(filesize)
+        sizes.append(size)
+        total_size+=size
+
+    logger.log(35,"Timestamp %f Files %d TotalSize %d",time.time(),nfiles,total_size)
+
+    # create the test files
+    for size in sizes:
+        create_hashfile(d,size=size)
 
+    time0=time.time()
```

Comment:
for future reference, wrapping time-measuring sections of the code using Python's context managers to can make the code cleaner: http://preshing.com/20110924/timing-your-code-using-pythons-with-statement/